src/platforms/web/runtime/directives/model.js

```diff
@@ -39,7 +39,7 @@ const directive = {
         // this also fixes the issue where some browsers e.g. iOS Chrome
         // fires "change" instead of "input" on autocomplete.
         el.addEventListener('change', onCompositionEnd)
-        if (!isAndroid) {
+        if (typeof window.CompositionEvent === 'function') {
```

Comment:
Dectect this everytime is not a good idea.

Reply:
Refactor as your suggestion, thx : )